js/wine-pairing.js

```diff
@@ -0,0 +1,22 @@
+/*
+    This is specifically for the wine pairing page
+*/
+document.addEventListener("DOMContentLoaded", async (event) => {
+    const wineFamily = getWineFamilyPreference();
+
+    if(!wineFamily) {
+        alert(`No wine selection has been mande as yet`);
+        return;
+    }
+
+    const wines= await fetchWines(wineFamily)
+    console.log(wines)
+
+    // Populae the drop downlist with the list of wines from the API 
```

Comment:
This is the part I am not sure about. From my analysis of the Spoonacular API, I think it is easyer to have a manually selected array of wines to populate the select (dropdowns) as I did today. See current index.js. Let me know if you have something different in mind and let's have a recap meeting. Thank you!